apps/wallet/src/ui/app/components/menu/content/VerifyLedgerConnectionStatus.tsx

```diff
@@ -37,10 +37,6 @@ export function VerifyLedgerConnectionStatus({
     );
     const timeoutIdRef = useRef<number>();
 
-    useEffect(() => {
```

Comment:
I read asymmetric effects were bad to do on Twitter from the React ppl 😨

Reply:
Does this mean we never clear the timeout though?

Reply:
yeah but I took Dan's word as god 😆 

https://twitter.com/dan_abramov/status/1648713546803412993